main.cpp

```diff
@@ -2,7 +2,54 @@
 
 using namespace std;
 
+void menuShow();
+void menuChoose();
+
 int main() {
 	cout << "Hello fellow Students!" << endl;
+    menuShow();
+    menuChoose();
```

Comment:
wyjscie z programu nie pelni aktualnie zadnej fukncji poniewaz po pierwszym wybraniu opcji i jej wykonaiu program sie zakonczy. Rozwaz uzycie petli while.

Reply:
while {...} (sign !=0) ?

Reply:
sign troche tutaj nie pasuje poniewaz nie przekazujemy znaku a wybieramy opcje z menu, rozwaz zmiane nazwy tej zmiennej.
Co do petli while to jest forma wyglada tak:
while(option != exit_option) {...}
ewentulanie:
do {...} while (option != exit_option)